Makefile

```diff
@@ -82,6 +82,68 @@ pyinstaller: clean
 build-dmg: needs-version
 	$(PYTHON_EXEC) -m dmgbuild -s build_config/dmgbuild_settings.py "Kolibri ${KOLIBRI_VERSION}" dist/kolibri-${KOLIBRI_VERSION}.dmg
 
+.PHONY: webview2
+# Download WebView2 runtime installer
+webview2:
+	@if [ ! -f MicrosoftEdgeWebView2RuntimeInstallerX64.exe ]; then \
+		echo "Downloading WebView2 full installer..."; \
+		( \
+			trap 'echo "Interrupted. Cleaning up..."; rm -f MicrosoftEdgeWebView2RuntimeInstallerX64.exe; exit 1' INT TERM; \
+			wget https://go.microsoft.com/fwlink/?linkid=2124701 -O MicrosoftEdgeWebView2RuntimeInstallerX64.exe || { \
+				echo "\Download failed. Cleaning up..."; \
+				rm -f MicrosoftEdgeWebView2RuntimeInstallerX64.exe; \
+				exit 1; \
+			} \
+		); \
+	else \
+		echo "WebView2 full installer already present."; \
+	fi
+
+.PHONY: nssm
+# Download NSSM for Windows service management
+nssm:
+	@if [ ! -f nssm.exe ]; then \
+		echo "Downloading NSSM..."; \
+		( \
+			trap 'echo "Interrupted. Cleaning up..."; rm -f nssm.zip; rm -rf nssm; exit 1' INT TERM; \
+			mkdir -p nssm && \
+			wget https://nssm.cc/release/nssm-2.24.zip -O nssm.zip || { \
```

Comment:
Would be good to define nssm version as a Makefile variable at the top of the file so we can reference it in multiple places (and then update it in one place in future, just in case they do ever release a new version!)